### ISSUE TITLE
Add CDN installation example

### ISSUE THREAD

**Issue by asvae** [epicmaxco/vuestic-ui#835]
element has an example, where they add framework to the html page directly via CDN.
Guess that would be nice to have.

**Comment by smellyshovel** [comment]
Do we have a CDN minified build at the first place?

**Comment by asvae** [comment]
Also seems ready for implementation for me, just needs affirmation and a champion 😬.

**Comment by m0ksem** [comment]
Blocked by https://github.com/epicmaxco/vuestic-ui/issues/2825
